townService/testData/indoors.json

```diff
@@ -432,7 +432,7 @@
                  "y":1105.33333333333
                 }, 
                 {
-                 "type":"ConversationArea",
+                 "type":"PresentationArea",
```

Comment:
It seems like this is the foyer table, is there any reason this is a presentation area?

Reply:
I put it like that because I just didn't bother to add a new area for my tests. If we keep the indoors.json from main then it will be good.